kenshin-profile-app/kenshin-profile-app/SNSView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SNS.swift
+//  kenshin-profile-app
+//
+//  Created by miyamotokenshin on R 7/06/09.
+//
+
+import SwiftUI
+
+struct SNSView: View {
+    var body: some View {
+        NavigationStack {
+            List {
+                Label(
+                    title: {
+                        Text("kenshin0413")
+                            .font(.title2)
+                            .fontDesign(.serif)
+                            .padding()
+                    },icon: {
+                        Image("github")
+                            .resizable()
+                            .frame(width: 50, height: 50)
```

Comment:
基本画像を使うときはresizableと`.scaledToFit()`をセットで使うよ
`.scaledToFit()`は縦横比を固定することができる

```suggestion
                            .resizable()
                            .scaledToFit()
                            .frame(width: 50, height: 50)
```